jupyter_book/pdf.py

```diff
@@ -47,21 +72,101 @@ def update_latex_documents(latex_documents, latexoverrides):
     """
     Apply latexoverrides from _config.yml to latex_documents tuple
     """
-    latexdocs_tuple = (
-        "startdocname",
-        "targetname",
-        "title",
-        "author",
-        "theme",
-        "toctree_only",
-    )
+    if len(latex_documents) > 1:
+        _message_box(
+            "Latex documents has been specified as a multi element list in the _config",
+            "This suggests the user has made custom settings to their build",
+            "[Skipping] update_latex_documents for specific latex overrides",
+        )
+        return latex_documents
+    else:
+        # Extract latex_documents tuple
+        latex_documents = latex_documents[0]
+    # Apply single overrides from _config.yml
     updated_latexdocs = []
-    for loc, item in enumerate(latexdocs_tuple):
+    for loc, item in enumerate(LATEX_DOCUMENTS):
```

Comment:
@AakashGfude it is used here to iterate through the expected `tuple` elements based on that version spec

Reply:
yes. and then update those items in the tuple which are in the latexoverrides.

Reply:
so feel free to remove this if you think there is a better way to implement -- that is fine by me.